form/data_transfer_objects.rst

```diff
@@ -0,0 +1,347 @@
+.. index::
+   single: Form; Data Transfer Objects
+
+How to use Data Transfer Objects (DTOs)
+=======================================
+
+Data Transfer Objects can be used by forms to separate entities from the
+validation logic of forms.
+Entities should always have a valid state.
+When entities are used as data classes for a form, the data is injected into
+the entity and validated.
+When the validation fails, the invalid data is still left in the entity.
+This can lead to invalid data being saved in the database.
+
+You will use the Maker bundle to highlight the differences between using DTOs
```

Comment:
`You` should maybe be `We`?

Reply:
In [forms.rst](https://github.com/symfony/symfony-docs/blob/master/forms.rst), you is being used.
I think it's the better choice, as it sounds more empowering. The reader builds, we just teach them how.

Reply:
this sentence sounds strange, "You will etc"...: What? No I won't! :)
Maybe: 
Let's use the Maker bundle to highlight the differences between using DTOs and entities.